docs/jms.md

```diff
@@ -0,0 +1,108 @@
+## How to configure JMS for WildFly using out of process ActiveMQ
+
+### 1. Download ActiveMQ
+
+[Download](http://activemq.apache.org/activemq-5141-release.html) the version 5.14.1. and run the script ```ACTIVEMQ_LOCATION/bin/activemq start``` (needs superuser rights).
```

Comment:
FYI - tady stačí používat jeden backtick, tři jsou když to potřebuješ na víc řádek ;)